lizmap/modules/lizmap/lib/Commands/WMTSSeed.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Lizmap\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WMTSSeed extends \Jelix\Scripts\ModuleCommandAbstract
+{
+    protected function configure()
+    {
+        $this
+            ->setName('wmts:cache:seed')
+            ->setDescription('Generate cache for a layer')
+            ->setHelp('verbose option will provide more info')
+            ->addArgument('repository', InputArgument::REQUIRED, 'The repository ID')
+            ->addArgument('project', InputArgument::REQUIRED, 'The project name')
+            ->addArgument('layers', InputArgument::REQUIRED, 'Coma separated layers list or \'*\' ')
+            ->addArgument('TileMatrixSet', InputArgument::REQUIRED, 'the TileMatrixSet (CRS) for which you want to generate cache')
+            ->addArgument('TileMatrixMin', InputArgument::REQUIRED, 'the min zoom level to generate')
+            ->addArgument('TileMatrixMax', InputArgument::REQUIRED, 'the max zoom level to generate')
+            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'will do not generate cache, output statistics')
```

Comment:
```suggestion
            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'will not generate the cache, only output statistics')
```
? Mainly the "will not do"